File: Data Structures & Algorithms/two-integer-sum/submission-7.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> mp=new HashMap<>();
        int ans[]=new int[2];
        for(int i=0;i<nums.length;i++){
            mp.put(nums[i],i);
        }

        for(int i=0;i<nums.length;i++){
            int diff=target-nums[i];
            if(mp.containsKey(diff) && mp.get(diff)!=i){
                ans[0]=i;
                ans[1]=mp.get(diff);
            }
        }
        Arrays.sort(ans);
        return ans;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-7.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        if not nums:
            return []

        mp={}
        ans=[]
        for val in nums:
            mp[val]=mp.get(val,0)+1

        for key,val in mp.items():
            if val>=k:
                ans.append(key)

        return ans      